src/cpp/core/include/core/FileInfo.hpp

```diff
@@ -38,7 +38,8 @@ class FileInfo
       : absolutePath_(), 
         isDirectory_(false), 
         size_(0), 
-        lastWriteTime_(0)
+        lastWriteTime_(0),
+        isSymlink_(false)
```

Comment:
Is this change meant to be included? (don't see it referenced anywhere)

Reply:
This was caught by ASAN; we need to explicitly initialize the member to `false` otherwise when we reference it (as evidently we do in some cases) we'll end up reading uninitialized memory.